stable/sentry/Chart.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
+description: Sentry is a cross-platform crash reporting and aggregation platform.
+name: sentry
+version: 0.1.0
+keywords:
+  - debugging
+  - logging
+sources:
+  - https://github.com/getsentry/sentry
+home: https://sentry.io/
+icon: https://a0wx592cvgzripj.global.ssl.fastly.net/_static/6571516f8aed42e4172c9c439ba814c6/getsentry/images/branding/png/sentry-glyph-black.png
+maintainers:
+  - name: Justin Garrison
+    email: justin@linux.com
```

Comment:
Please update your name to your GitHub handle, so folks can easily ping you on future work on this chart.